CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+## v0.31.0
+
+*March 13th, 2019*
+
```

Comment:
This release brings pubsub 2.0, limits the mempool size to 1GB (`max_txs_bytes`) and number of `/subscribe` WS clients (`max_subscription_clients`) and adds `/unsubscribe_all` endpoint to lite client. It also contains many smaller improvements and bugfixes. Pubsub 2.0 is an improved version of the older pubsub, which is a) non-blocking b) has nicer API. Note our HttpClient's interface got updated to reflect the pubsub changes and now also has a better API for WS subscriptions.